src/checkExistence.js

```diff
@@ -15,7 +15,10 @@
  * @return {boolean}
  */
 function checkExistence(array, element) {
-  // write code here
+  if (array.includes(element)) {
```

Comment:
includes return boolean value.
no need to write return `false or true`
```suggestion
  return array.includes(element)
```